lib/matplotlib/__init__.py

```diff
@@ -331,18 +331,14 @@ def impl(args, regex, min_ver=None, ignore_exit_code=False):
         raise ExecutableNotFoundError(message)
     elif name == "inkscape":
         try:
-            # use headless option first (works with inkscape version < 1.0):
-            info = impl(["inkscape", "--without-gui", "-V"],
-                        "Inkscape ([^ ]*)")
+            # Try headless option first (needed for Inkscape version < 1.0):
+            return impl(["inkscape", "--without-gui", "-V"],
+                         "Inkscape ([^ ]*)")
```

Comment:
One space too many; this line isn't in the list. I think it passes pep8 because it happens to be a multiple of 4, but that's strange to do here.

Reply:
yup